### DISCUSSION TITLE
Allow to set REST API headers

### DISCUSSION THREAD

**Discussion by hysh** [jsforce/jsforce#1443] [Ideas]
I was trying to update/create a case but ownerid was overwritten by the auto assign rule.
I added Sforce-Auto-Assign to FALSE in the header and seems to resolve this issue.

connection.js

```
   headers : {
     "Content-Type" : "application/json",
     'Sforce-Auto-Assign': 'FALSE'
   }
```

**Comment by cristiand391** [comment]
**EDIT**:

passing http headers to `conn.sobject` CRUD methods is supported, see:
https://jsforce.github.io/document/#http-headers
------------------------
moved to `ideas`, default value is `TRUE` but jsforce doesn't expose an option to toggle this.
https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/headers_autoassign.htm
